app/models/concerns/aggregatable.rb

```diff
@@ -60,19 +60,17 @@ class AggregateListError < StandardError; end
   end
 
   def add_item_from_child_list(item)
-    raise AggregateListError, 'add_item_from_child_list method only available on aggregate lists' unless is_aggregate_list?
+    raise AggregateListError.new('add_item_from_child_list method only available on aggregate lists') unless aggregate_list?
 
     list_items.combine_or_create!(public_list_item_attrs(item).merge('list_id' => id))
   end
 
   def remove_item_from_child_list(attrs)
-    raise AggregateListError, 'remove_item_from_child_list method only available on aggregate lists' unless is_aggregate_list?
+    raise AggregateListError.new('remove_item_from_child_list method only available on aggregate lists') unless aggregate_list?
 
     existing_item = list_items.find_by('description ILIKE ?', attrs['description'])
 
-    if existing_item.nil? || existing_item.quantity < attrs['quantity']
-      raise AggregateListError, 'item passed to remove_item_from_child_list method is not represented on the aggregate list'
-    end
+    raise AggregateListError.new('item passed to remove_item_from_child_list method is not represented on the aggregate list') if existing_item.nil? || existing_item.quantity < attrs['quantity']
```

Comment:
I can't decide if it's a good or bad thing that I didn't turn on the line length cop.